lib/src/test/kotlin/at/bitfire/vcard4android/ContactTest.kt

```diff
@@ -41,6 +43,21 @@ class ContactTest {
     }
 
 
+    @Test
+    fun testToString_TruncatesLargeFields() {
+        val c = Contact(
+            displayName = "Test",
+            members = mutableSetOf("1", "2", "3"),
+            emails = LinkedList(listOf(LabeledProperty(Email("test@example.com")))),
+            note = "Some Text\n".repeat(1000),
+            photo = ByteArray(10*1024*1024) { 'A'.toByte() },  // 10 MB
```

Comment:
IDE says using `toByte()` diretly is deprecated and suggests using `'A'.code.toByte()` instead.